reporter/jsonConsole.go

```diff
@@ -183,7 +190,7 @@ func (c *jsonConsole) ScenarioEnd(scenario *gauge.Scenario, res result.Result, i
 		Res: &executionResult{
 			Status:            getScenarioStatus(res.(*result.ScenarioResult)),
 			Time:              res.ExecTime(),
-			Errors:            getErrors(getAllStepsFromScenario(res.(*result.ScenarioResult).ProtoScenario), i.CurrentSpec.FileName),
+			Errors:            getErrors(c, getAllStepsFromScenario(res.(*result.ScenarioResult).ProtoScenario), i.CurrentSpec.FileName, i),
```

Comment:
Is there any specific reason to pass the JSONConsole instead of passing the cache?

Reply:
No. We can pass `stepCache` instead of `JSONCosole`.